### ISSUE TITLE
PacMan readme

### ISSUE THREAD

**Issue by tacertain** [JimmyStones/Hiscores_MiSTer#9]
In the README.txt for PacMan, https://github.com/MiSTer-devel/Arcade-Pacman_MiSTer/blob/master/README.txt, it says that you need to reload the core after turning on autosave. That's no longer true with v14, right? Just wanted to make sure before submitting a PR.

**Comment by JimmyStones** [comment]
You are correct, that note can be removed if it is using the latest hiscore module
